system/reloc64.c

```diff
@@ -70,12 +72,21 @@ typedef struct
 static inline Elf64_Addr __attribute__ ((unused)) get_load_address(void)
 {
     Elf64_Addr addr;
+#if defined(__i386__) || defined(__x86_64__)
```

Comment:
`__x86_64__` alone would suffice here, since this file is used only there.

Reply:
OK, I will adjust next time.